cmd/iop/install.go

```diff
@@ -46,10 +46,33 @@ func installManifests(args *rootArgs) {
 
 	manifests, err := genManifests(args)
 	if err != nil {
-		log.Fatalf(err.Error())
+		logAndPrintf(args, "%s", err)
+		os.Exit(1)
+	}
+
+	out, err := manifest.ApplyAll(manifests, version.NewVersion("", 1, 2, 0, ""), args.dryRun, args.verbose)
+	if err != nil {
+		logAndPrintf(args, "%s", err)
+		os.Exit(1)
 	}
 
-	if err := manifest.ApplyAll(manifests, version.NewVersion("", 1, 2, 0, ""), args.dryRun, args.verbose); err != nil {
-		log.Fatalf(err.Error())
+	for cn := range manifests {
+
+		cs := fmt.Sprintf("CompositeOutput for component %s:", cn)
+		logAndPrintf(args, "\n%s\n%s", cs, underline(cs, "="))
+		if out.Err[cn] != nil {
+			logAndPrintf(args, "Errors: %s\n", out.Err[cn])
+		}
+		if strings.TrimSpace(out.Stderr[cn]) != "" {
+			logAndPrintf(args, "Error strings:\n%s\n", out.Stderr[cn])
+		}
+		if strings.TrimSpace(out.Stdout[cn]) != "" {
+			logAndPrintf(args, "Command output:\n%s\n", out.Stdout[cn])
+		}
 	}
 }
+
+// underline generates an underline with the underlineChar of length s.
+func underline(s, underlineChar string) string {
```

Comment:
I'm not sure this warrants its own function. `strings.Repeat("=", len(cs))` is equally readable IMO.